src/harness/testcases/WINNF_FT_S_REG_testcase.py

```diff
@@ -0,0 +1,262 @@
+#    Copyright 2016 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+import json
+import os
+import unittest
+
+import sas
+from util import winnforum_testcase
+
+
+class RegistrationTestcase(unittest.TestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_REG_1(self):
+    """Array Multi-Step registration for CBSDs (Cat A and B).
+
+    The response should be SUCCESS.
+    """
+
+    # Load Devices
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+
+    # Pre-load conditionals
+    conditionals_a = {
+        'cbsdCategory': device_a['cbsdCategory'],
+        'fccId': device_a['fccId'],
+        'cbsdSerialNumber': device_a['cbsdSerialNumber'],
+        'airInterface': device_a['airInterface'],
+        'installationParam': device_a['installationParam']
```

Comment:
Do we need to load measCapability into the SAS-under-test when we are loading the conditional parameters?  A value of "measCapability": [] is an accepted value indicating that the CBSD has meascapability="none".  I'm reading the latest version of part 1 (released 10/13/2017) and it says "All REGConditional
parameters for the three CBSDs (a combination of both Cat A and
Cat B) shall already be pre-loaded into the SAS" so it seems like we should load meascapability into the SAS-under test even if that value is a blank value which means none.......the end result should be the same but if we want to fully follow the instructions we may want to load a 'measCapability': [ ] along wiht the other ocnditionals...thoughts?

Reply:
Agreed, Done.